Tests/KafkaTests/Utilities.swift

```diff
@@ -13,6 +13,7 @@
 //===----------------------------------------------------------------------===//
 
 import Logging
+import Metrics
```

Comment:
Do we still need this here?

Reply:
Surely not! Thank you for catching that :)